### PR TITLE
tricycle drive motorController debugged

### PR DESCRIPTION
**Pull request by todorangrg**
Now the tricycle motor controller behaves as expected, even when acceleration limits are low.

### CODE REVIEW

**Comment by hsu** [comment]
Please add a test or point to launch-ed world scenario that we can use to verify this bug fix? Thanks.

**Comment by maxbader** [comment]
A working world scenario can be fount in tricycle_drive_scenario.launch

File: gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp [hunk 1]
```diff
@@ -222,17 +222,15 @@ void GazeboRosTricycleDrive::motorController ( double target_speed, double targe
...
         double diff_speed = current_speed - target_speed;
         if ( fabs ( diff_speed ) < wheel_speed_tolerance_ ) {
-            applied_speed = target_speed;
-        } else if ( diff_speed < target_speed ) {
-            applied_speed = current_speed + wheel_acceleration_ * dt;
-        } else {
-            applied_speed = current_speed - wheel_deceleration_ * dt;
+            applied_speed = current_speed;
+        } else if ( fabs(diff_speed) > wheel_acceleration_ * dt ) {
+            if(diff_speed > 0){ applied_speed = current_speed - wheel_acceleration_ * dt;}
```
Comment (hsu) [inline gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp:232]: Please use the same bracket indent convention as previous code:

```
           if (diff_speed > 0) {
             applied_speed = current_speed - wheel_acceleration_ * dt;
           }
```

**Comment by hsu** [comment]
moving to #444, please comment there and I'll update that pr. Thanks.

**Pull request closed without merging**